static/app/views/replays/deadRageClick/selectorTable.tsx

```diff
@@ -25,27 +27,35 @@ export function getAriaLabel(str: string) {
   return pre.substring(0, pre.lastIndexOf('"]'));
 }
 
-export function hydratedSelectorData(data, clickType): DeadRageSelectorItem[] {
-  return data
-    .filter(d => d[clickType] > 0)
-    .map(d => {
-      return {
-        [clickType]: d[clickType],
-        dom_element: d.dom_element,
-        element: d.dom_element.split(/[#.]+/)[0],
-        aria_label: getAriaLabel(d.dom_element),
-      };
-    });
+export function hydratedSelectorData(data, clickType?): DeadRageSelectorItem[] {
+  return clickType
+    ? data.map(d => {
+        return {
+          [clickType]: d[clickType],
+          dom_element: d.dom_element,
+          element: d.dom_element.split(/[#.]+/)[0],
+          aria_label: getAriaLabel(d.dom_element),
+        };
+      })
+    : data.map(d => {
+        return {
+          count_dead_clicks: d.count_dead_clicks,
+          count_rage_clicks: d.count_rage_clicks,
+          dom_element: d.dom_element,
+          element: d.dom_element.split(/[#.]+/)[0],
+          aria_label: getAriaLabel(d.dom_element),
+        };
+      });
```

Comment:
l: You could do something like this so that you don't have to keep the two objects in sync:

```suggestion
  return data.map(d => ({
    ...(clickType ? {[clickType]: d[clickType]} : {
      count_dead_clicks: d.count_dead_clicks,
      count_rage_clicks: d.count_rage_clicks,
    }),
    dom_element: d.dom_element,
    element: d.dom_element.split(/[#.]+/)[0],
    aria_label: getAriaLabel(d.dom_element),
  }));  
```